runtime/compiler/runtime/MetaData.cpp

```diff
@@ -1731,3 +1821,202 @@ createMethodMetaData(
 
    return data;
    }
+
+#if defined(J9VM_OPT_MICROJIT)
+//
+//the routine that sequences the creation of the meta-data for 
+//   a method compiled for MJIT
+//
+TR_MethodMetaData *
+createMJITMethodMetaData(
+   TR_J9VMBase & vmArg,
+   TR_ResolvedMethod *vmMethod,
+   TR::Compilation *comp
+   )
+   {
+   //---------------------------------------------------------------------------
+   //This function needs to create the meta-data structures for: 
+   //   GC maps, Exceptions, inlining calls, stack atlas, 
+   //   GPU optimizations and internal pointer table
+   TR_J9VMBase * vm = &vmArg;
+   MJIT_ExceptionTableEntryIterator exceptionIterator(comp);
+   TR::ResolvedMethodSymbol * methodSymbol = comp->getJittedMethodSymbol();
+   TR::CodeGenerator * cg = comp->cg();
+   TR::GCStackAtlas * trStackAtlas = cg->getStackAtlas();
+
+   // --------------------------------------------------------------------------
+   // Find unmergeable GC maps
+   //TODO: Create mechanism like treetop to support this at a MicroJIT level.
```

Comment:
Suggestion: change "TODO:" into "TODO: Microjit:" so that we can search later for all the left over items for microjit in particular.